desktop/windows/release-notes/index.md

```diff
@@ -17,12 +17,43 @@ redirect_from:
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+## Docker Desktop 4.2.0
+2021-11-xx
+
+> Download Docker Desktop
+>
+> [For Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){: .button .primary-btn }
+
+### New
+- **Pause/Resume**: Save your battery with Docker Pause / Resume! For more information, see [Docker Pause](https://docs.docker.com/go/desktop-pause/).
+- **Software Updates**: More clear settings section as all update related settings are moved to update section and now also available for free tier users. The update popup to prompt for a new version has been removed.
+- **Window Management**: Dashboard window size/position is persisted/restored on closing/opening.
+
+### Upgrades
+
+- [Docker Engine v20.10.10](https://docs.docker.com/engine/release-notes/#201010)
+- [containerd v1.4.11](https://github.com/containerd/containerd/releases/tag/v1.4.11)
+- [runc v1.0.2](https://github.com/opencontainers/runc/releases/tag/v1.0.2)
+- [Go 1.17.2](https://golang.org/doc/go1.17)
+- [Compose CLI v2.1.0](https://github.com/docker/compose-cli/tree/v1.0.18)
+- docker-scan 0.9.0
+
+### Bug fixes and minor changes
+
+- Improved: Self-diagnose now also checks for overlap between host IPs and `docker networks`.
+- Fixed position of update available indicator in dashboard window header.
+- Fixed Docker Desktop sometimes hanging when clicking Exit in the fatal error dialog.
+- Fixed download update popup coming back every day when an update has been downloaded but not applied yet [docker/for-win#12188](https://github.com/docker/for-win/issues/12188).
+- Fixed installing a new update killing the application before it has time to shut down.
```

Comment:
```suggestion
- Fixed an issue that caused the installation of a new update to crash Docker Desktop before shutting it down.
```

Not sure if that makes sense, especially the shutting down part. Could you provide more info pls?

Reply:
@mat007 could you clarify this? I think you were the author of the PR.

Reply:
It was not crashing it, only leaving a icon in the tray which went away when hovering it with the mouse.
That and probably sometimes breaking the WSL 2 integration 😬

Reply:
@joe0BAB Updated the suggestion on this note. PTAL